src/test/java/seedu/address/logic/commands/TagInfoCommandTest.java

```diff
@@ -0,0 +1,45 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalDate.getTypicalCountdown;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.Shortcut;
+import seedu.address.model.UserPrefs;
+
+public class TagInfoCommandTest {
+
+    private Model model;
+
+    @BeforeEach
+    public void setUp() {
+        model = new ModelManager(getTypicalAddressBook(), getTypicalCountdown(), new UserPrefs(), new Shortcut());
+    }
+
+
+    @Test
+    public void executeEmptyAddressBookSuccess() {
```

Comment:
I think the name of the test method should be `execute_EmptyAddressBook_Success`(use underscore), this applies to all your method name below.